types/react/ts5.0/index.d.ts

```diff
@@ -3758,6 +3759,159 @@ declare namespace React {
         zoomAndPan?: string | undefined;
     }
 
+    // https://www.w3.org/TR/mathml-core/#global-attributes
+    //
+    // The approach taken with all MathMLElement attribute name casing is the following:
+    //
+    // * Names of attributes for the base MathMLAttributes that have special casings in other elements types use those same casings
+    // * Otherwise, the casing of attributes match that of the mathml-core specification
+    //
+    interface MathMLAttributes extends DOMAttributes<MathMLElement> {
+        className?: string | undefined;
+        dir?: "ltr" | "rtl" | undefined;
+        displaystyle?: "false" | "true" | undefined;
+        href?: string | undefined;
+        id?: string | undefined;
+        nonce?: string | undefined;
+        scriptlevel?: string | undefined;
+        style?: CSSProperties | undefined;
+        tabIndex?: number | undefined;
+        // This attributes are considered legacy but still described in the specification:
+        //
+        // https://www.w3.org/TR/mathml-core/#legacy-mathml-style-attributes
+        //
+        mathbackground?: string | undefined;
+        mathcolor?: string | undefined;
+        mathsize?: string | undefined;
+    }
+
+    // MathML elements and attributes are described here:
+    //
+    // https://www.w3.org/TR/mathml-core/#mathml-elements-and-attributes
+    //
+    interface MathMLMActionAttributes extends MathMLAttributes {
+        actiontype?: string | undefined;
+        selection?: string | undefined;
+    }
+    interface MathMLMathAttributes extends MathMLAttributes {
+        display?: "block" | "inline" | undefined;
+    }
+    interface MathMLMErrorAttributes extends MathMLAttributes {}
+    interface MathMLMFracAttributes extends MathMLAttributes {
+        linethickness?: string | undefined;
+    }
+    interface MathMLMIAttributes extends MathMLAttributes {
+        mathvariant?: "normal" | undefined;
+    }
+    interface MathMLMMultiscriptsAttributes extends MathMLAttributes {}
+    interface MathMLMNAttributes extends MathMLAttributes {}
+    interface MathMLMOAttributes extends MathMLAttributes {
+        /** This attribute is non-standard. */
+        accent?: boolean | undefined;
+        fence?: boolean | undefined;
```

Comment:
`<mo>` element missing `form` attribute.

```suggestion
        fence?: boolean | undefined;
        form?: "prefix" | "infix" | "postfix" | undefined;
```